backend/development.ini.sample

```diff
@@ -343,18 +343,18 @@ email.notification.from.default_label = Tracim Notifications
 ; email.notification.reply_to.email = reply+cid{content_id}@domainname.tld
```

Comment:
I think its better if this block of example is starting by `#` for each line, because if we uncomment this line, and uncomment same line in next block, we can have something wrong in configuration.
 
For me its better to replace `; email.notification.reply_to.email` by `# email.notification.reply_to.email` on this three preview line.

Reply:
I agree, following pattern permit to know system default value:

    ; config_name = default_value

Keep only one comment with `;` help to know what is the default value.

So i suggest to use  `#` for example list, and add config line with it's default value using `; `

Reply:
@PhilippeAccorsi do you agree with this ?
If yes, i can try to check now or change this for another later issue .

Reply:
Yes I agree with this.